lib/elixir/lib/kernel.ex

```diff
@@ -261,6 +261,8 @@ defmodule Kernel do
   @doc """
   Returns the head of a list. Raises `ArgumentError` if the list is empty.
 
+  It works with improper lists.
```

Comment:
@whatyouhide I cannot think of anything other than "This function accepts improper lists"